.github/workflows/main.yaml

```diff
@@ -91,14 +91,22 @@ jobs:
             development/kv/data/repox url | artifactory_url;
             development/kv/data/burgr github_username | burgrx_username;
             development/kv/data/burgr github_password | burgrx_password;
+      - name: Vault AWS Secrets
```

Comment:
can you be more specific and include `Vault Binaries AWS Secrets` ? Also update the `id` accordingly

We have other AWS secrets coming from SSO now and yeah I already had to rename a bit things herehttps://github.com/SonarSource/gh-action_release/compare/master...feat/BUILD-4602-Burgr-independant-releasability